app/assets/stylesheets/custom.scss

```diff
@@ -303,24 +312,37 @@ ul, menu, dir {
   font-size: 40px;
 }
 
-.listing-show-page {
-  margin-bottom: 20px;
-  padding: 20px;
-}
-
 .listing-header {
   margin-left: 50px;
   margin-top: 1px;
 }
 
 .user-show-row {
-  background: #c5eaea;
   margin-left: 50px;
   margin-right: 50px;
   margin-top: 10px;
   padding: 20px;
 }
 
+.profile-header {
+  background: linear-gradient(rgba(240, 240, 230, 0.6), rgba(255, 255, 255, 0.6)), url(/assets/form-pattern.png) repeat;
+}
+
+.user-show-name {
+margin-left: 60px;
+font-size: 40px;
+}
+
+.user-image {
+  margin-left: 73px;
+}
+
+.about-me {
+  margin-left: 20px;
+  background: #faf8f2;
```

Comment:
Color literals like `#faf8f2` should only be used in variable declarations; they should be referred to via variable everywhere else.